arlo-client/src/components/Audit/Setup/Participants/index.tsx

```diff
@@ -112,24 +112,35 @@ const Participants: React.FC<IProps> = ({ locked, nextStage }: IProps) => {
               </FormSectionDescription>
             </FormSection>
             <FormSection>
-              <FileInput
-                inputProps={{
-                  accept: '.csv',
-                  name: 'csv',
-                }}
-                onInputChange={e => {
-                  setFieldValue(
-                    'csv',
-                    (e.currentTarget.files && e.currentTarget.files[0]) ||
-                      undefined
-                  )
-                }}
-                hasSelection={!!values.csv}
-                text={values.csv ? values.csv.name : 'Select a CSV...'}
-                onBlur={handleBlur}
-              />
-              {errors.csv && touched.csv && (
-                <ErrorLabel>{errors.csv}</ErrorLabel>
+              {isEditing || !file ? (
```

Comment:
for clarity, should this have `()` around the or'ed conditions?

Reply:
@benadida I actually just tried to do this, and the style linter removed them and prevented the empty commit. Upon reflection, the only time we put parentheses around stuff like this is when there's multiple instances of `||` or `&&` that need to be separated.

Reply:
ok, I can live with it, it's just my low-key-OCD is triggering ;)